apps/settings/js/panels/simpin/simpin.js

```diff
@@ -126,15 +127,23 @@ define(function(require) {
       cardIndex = parseInt(cardIndex, 10);
 
       switch (type) {
-        case 'checkSimPin':
```

Comment:
ooh I did not notice the difference between 'change' and `click` event, thanks for update.